nixos/modules/installer/tools/nixos-option/default.nix

```diff
@@ -0,0 +1,7 @@
+{stdenv, boost, cmake, pkgconfig, nix, ... }:
+stdenv.mkDerivation rec {
+  name = "nixos-option";
+  src = ./.;
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ boost nix ];
+}
```

Comment:
`meta.license` should be added. In this case, because code from Nix is taken (LGPL 2.1).

Would it make sense to unpack the Nix source and copy it from there or does that make it too complex?

Reply:
License added.

Unpacking and copying from nix sources: That would probably be more more trouble than it's worth -- more code to do that than is copied over.  It's not critical that it be kept in perfect sync with nix -- these are convenience utility-style functions, not things that depend upon nix implementation details in a brittle way.  I plan to propose to the nix maintainers making these utility functions part of nix's exported API, but I wanted to move forward with this in parallel.  Ideally, this copied-from-nix code will go away soon.

Thanks!